images/scripts/utils.sh

```
@@ -178,13 +178,23 @@ arkd-init(){
   arkd-sim wallet unlock --password ark
   waitForArkdToSync
 
-  bitcoin-cli-sim-server -rpcwallet=regtest sendtoaddress $(arkd-sim wallet address) 25
+  bitcoin-cli-sim-server -rpcwallet=regtest sendtoaddress $(arkd-sim wallet address) 100
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)